docs/src/what-is-docsearch.md

```diff
@@ -5,10 +5,10 @@ title: What is DocSearch?
 
 ## Why?
 
-DocSearch is born out of the need to scratch our own itch. As developers, we
-spent a lot of time reading documentation, and it can be hard to find relevant
-information in large documentations. We're not blaming anyone here; building a
-good search is a challenge.
+We have created DocSearch because we used to scratch our own itch. As
```

Comment:
```suggestion
We have created DocSearch because we are scratching our own itch. As
```

We aren't really using DS anymore for many Algolia projects, but it's still for us in the first place no?

Reply:
Because we are going way beyond. This sentence states the need and global feeling not the solution. I will keep it. Thanks